en/simulation/multi-vehicle-simulation.md

```diff
@@ -12,11 +12,11 @@ Tools/gazebo_sitl_multiple_run.sh [-m <model>] [-n <number_of_vehicles>]
 ```
 
 - `<model>`: The vehicle type/model to spawn: `iris` (default), `plane`.
-- `number_of_vehicles`: The number of vehicles to spawn. Default is 3. Maximum is 10.
+- `number_of_vehicles`: The number of vehicles to spawn. Default is 3. Maximum is 255.
 
 Each vehicle instance is allocated a unique MAVLink system id (1, 2, 3, etc.) and can be accessed from a unique remote offboard UDP port (14540, 14541, 14542, etc.).
```

Comment:
@Jaeyoung-Lim I'll merge -thanks for the change! 

Question, the offboard port is 14540+ as it says here still? Is the QGC port still the same - 14550 - ie same access port to pick up all the vehicles from QGC [I assume so, just being thorough]